.changeset/loud-cameras-occur.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rspress/plugin-auto-nav-sidebar": minor
+"@rspress/theme-default": minor
+"@rspress/shared": minor
+---
+
+support overviewDepth to control the minimum title displayed in the overview page
```

Comment:
Can we have some E2E cases for this new feature?

Reply:
done